src/test/java/org/frankframework/frankdoc/testtarget/examples/becomes/mandatory/Master.java

```diff
@@ -0,0 +1,16 @@
+package org.frankframework.frankdoc.testtarget.examples.becomes.mandatory;
```

Comment:
Zou je hier 'becomes.mandatory' kunnen vervangen door 'ignore.in.compatibility'?

Reply:
Gedaan.